reporters/package.json

```diff
@@ -16,6 +16,7 @@
     "node-fetch": "^2.3.0"
   },
   "homepage": "https://umaproject.org",
+  "main": "index.js",
```

Comment:
Unrelated, this package was missing a `main` prop which is the entry point for reporters